src/Soil-Core-Tests/SoilIndexTest.class.st

```diff
@@ -92,6 +92,35 @@ SoilIndexTest >> testConcurrentRemoveKey [
 
 ]
 
+{ #category : #tests }
+SoilIndexTest >> testFlushIndexPages [
+	| tx dict capacity txn2 txn1 root1 root2 |
+	tx := soil newTransaction.
+	dict := SoilSkipListDictionary new
+		keySize: 16;
```

Comment:
I will (but after this is merged) check to make it to work for both BTree and SkipList